greet/greet.go

```diff
@@ -1,8 +1,14 @@
 package greet
 
-import "fmt"
+import (
+	"fmt"
+	"log"
+)
 
 // Greet function returns greeting for given name
 func Greet(name string) string {
+	if len(name) < 1 {
+		log.Panic("Name must be more than 0 characters long")
```

Comment:
Instead of panicking, you should return an error from this function and let the caller of the function decide what to do with the error.